src/PrestaShopBundle/Controller/Admin/Configure/AdvancedParameters/ImportController.php

```diff
@@ -115,6 +115,8 @@ public function importAction(Request $request)
     /**
      * Handle import file upload via AJAX, sending authorization errors in JSON.
      *
+     * @AdminSecurity("is_granted('create', request.get('_legacy_controller')) || is_granted('update', request.get('_legacy_controller'))")
```

Comment:
From the manual check under it seems like `delete` permission should also be added

Reply:
Same as above: I now understand why AdminSecurity was not used here. Response must be JSON. AdminSecurity does not handle that.